spec/rubocop/cop/naming/constant_name_spec.rb

```diff
@@ -25,6 +25,14 @@
     RUBY
   end
 
+  it 'registers 1 offense if rhs is offending const assignment' do
+    inspect_source(<<-RUBY.strip_indent)
+      Bar = Foo = 4
+    RUBY
+    expect(cop.offenses.size).to eq(1)
+    expect(cop.messages).to eq(['Use SCREAMING_SNAKE_CASE for constants.'])
```

Comment:
`expect_offenses` did not work with this example, so I adopted `inspect_source` to show an offense is still raised in this scenario.

Reply:
Did you mean `expect_offense` (without the `s`)?

This worked for me. 
```ruby
  it 'registers 1 offense if rhs is offending const assignment' do
    expect_offense(<<-RUBY.strip_indent)
      Bar = Foo = 4
            ^^^ Use SCREAMING_SNAKE_CASE for constants.
    RUBY
  end
```

Reply:
Yes. Thank you!